pages/formulaire/nps.tsx

```diff
@@ -16,7 +20,7 @@ export const visitorTypes = [
     label: 'Dirigeant(e) d’entreprise ou d’association',
   },
   {
-    value: 'Agent public',
+    value: 'Agent public non connecté',
     label: 'Agent public',
   },
```

Comment:
If we change it we should also change https://github.com/etalab/annuaire-entreprises-site/blob/1b8eeedee168cd46c77c0c202d56725eb459ce0d/clients/matomo/index.ts#L234-L235

to take into account every userType starting with `agent` to be considered as Agents Publics in the stats other wise it will break history